src/mmw/mmw/settings/layer_settings.py

```diff
@@ -145,6 +145,30 @@
         # overlaps with perimeter polygon.
     },
     {
+        'code': 'nhd_quality_tp',
+        'display': 'Delaware River Basin TP Concentration',
```

Comment:
I considered that as well, but that would still leave us in a situation of rendering blue lines for DRB streams that don't have water quality records, which I don't think they want.